File: 최혜린/programmers/Lv2/01.JadenCase 문자열 만들기.js
function solution(s) {
  let arr = s.split(' ');

  let res = arr.map(x => x.charAt(0).toUpperCase() + x.slice(1).toLowerCase()).join(" ");

  return res.toString();
}

s = "3people unFollowed me";
console.log(solution(s));

File: 최혜린/programmers/Lv2/01.JadenCase 문자열 만들기.py
def solution(s):
    arr = s.split(' ')
    ans = []
    
    for i in arr:
        if i:
            ans.append(i[0].upper() + i[1:].lower())
        else:
            ans.append(i)
        
    return ' '.join(ans)

# title, capitalize 내장함수 사용한 코드
def solution(s):
    ans = ''
    arr = s.split(' ')

    for i in range(len(arr)):
        # 한 단어 기준 무조건 첫 문자만 대문자로 처리
        arr[i] = arr[i].capitalize()
    
    return ' '.join(arr)

s = "for the last week"
print(solution(s))